concordium-node/build.rs

```diff
@@ -157,7 +159,10 @@ fn link_ghc_libs() -> std::io::Result<std::path::PathBuf> {
             "--print-libdir",
         ]))
     });
+    #[cfg(all(not(feature = "static"), target_os = "linux"))]
     let rts_dir = Path::new(&ghc_lib_dir).join("rts");
+    #[cfg(all(not(feature = "static"), target_os = "macos"))]
+    let rts_dir = Path::new(&ghc_lib_dir).join(GHC_VARIANT);
```

Comment:
Please document your findings here so that future maintainers can act.

That is, document that for the M1 GHC distribution the shared libraries for the RTS are in the GHC_VARIANT directory, not where they are for x86 distributions.

Reply:
isn't the code explaining what you are telling above?

Reply:
Also, this difference is valid for Intel macs too.